gitpod-io/gitpod
# move not running overlay to the supervisor

#### How to test
- stop a workspace and check that loading screen is shown with stopped state and start button
- try to restart the workspace from the loading screen as well as from the dashboard, check that the tab with stopped workspace gets reloaded in both cases
- check that each workspace tab has only one connection to the gitpod server (if not reconnections happen)

#### Out of scope
- testing out of credit alerts
- moving credit notifications from Theia
